vignettes/miaSim.Rmd

```diff
@@ -96,7 +99,8 @@ The number of species specified in the interaction matrix must be the same as th
 Hubbell Neutral simulation model characterizes diversity and relative abundance of species in ecological communities assuming migration, births and deaths but no interactions. Losses become replaced by migration or birth.
 
 ```{r}
-hubbellmodel <- simulateHubbell(n_species = 8, M = 10, I = 1000, d = 50,m = 0.02, tend = 100)
+hubbellmodel <- simulateHubbell(n_species = 8, M = 10, carrying_capacity = 1000,
+                                k_events = 50, migration_p = 0.02, t_end = 100)
```

Comment:
in general it would be more standard and perhaps also more intuitive in examples to use, instead of "hubbellmodel", names like x, mat, tse etc that refer to the type of the object. Kindly consider this here and elsewhere.